syntax = "proto3";

option java_multiple_files = true;
option java_package = "ru.itis.max";

package max;

service MaxCalculator {
  rpc calculate_max(stream Number) returns (stream Number) {}
}

message Number {
  int64 value = 1;
}

